etc/xsd/assembling.xsd

```diff
@@ -125,5 +125,6 @@ SPDX-License-Identifier: MPL-2.0 -->
   <xs:complexType name="MacroConnect">
     <xs:attribute name="macroConnection" use="required" type="xs:string"/>
     <xs:attribute name="id" use="required" type="xs:string"/>
+    <xs:attribute name="mandatory" type="xs:boolean"/>
```

Comment:
I am wondering if the `use="optional"` attribute would be necessary in that case?

Reply:
Hi, yes, the default value would be "false"